releases/release-1.31/release-notes/maps/pr-124828-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 124828
+releasenote:
+  text: Kubernetes is now built with go 1.22.3
```

Comment:
```suggestion
  text: Kubernetes was built with Go 1.22.3.
```

Reply:
@npolshakova is it ok to use 'past' and 'passive' times ?

Reply:
Id suggest "Built Kubernetes with Go 1.22.3" instead.
also interesting that there is this pr and then [releases/release-1.31/release-notes/maps/pr-125363-map.yaml](https://github.com/kubernetes/sig-release/pull/2542/files#diff-203b8a6eae06ce7cf5fd805f8598dffe015046b6fd76e48aa13dfcdd941bc9b2)